### PR TITLE
Update docker-images to version 72

### PR DESCRIPTION
**Pull request by hashhar**
## Release notes

(x) This is not user-visible or docs only and no release notes are required.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -69,7 +69,7 @@
         <dep.iceberg.version>1.0.0</dep.iceberg.version>
         <dep.spotbugs-annotations.version>4.7.2</dep.spotbugs-annotations.version>
 
-        <dep.docker.images.version>65</dep.docker.images.version>
+        <dep.docker.images.version>70</dep.docker.images.version>
```
Comment (nineinchnick) [inline pom.xml:72]: Note that #14569 also updates images, you may need same changes here

**Review by nineinchnick — APPROVED** [review]

**Comment by nineinchnick** [comment]
I ran some PTs using this branch on an M1 MacBook and they seem to work fine. For example, this one:
```
./testing/bin/ptl test run --environment singlenode-postgresql --config config-default --logs-dir logs/ --timeout 2h -- -g configured_features,postgresql
```
takes 1 minute 40 seconds and it can't even finish (timeouts) when run on master.

**Comment by hashhar** [comment]
hdp3, suite-azure seems like legitimate failure (or flakiness).

spark related things fail because Spark container's healthchecks fail - https://github.com/trinodb/docker-images/pull/147/files#r1007802848.

**Comment by hashhar** [comment]
Something weird with Spark - the env stays up and container remains healthy locally. Will see what happens when running a suite.

**Comment by hashhar** [comment]
I'll see if changing https://github.com/trinodb/docker-images/commit/3b5181fe125bbc14cbb3681819a6516857c59902#diff-31c91f5c3ac99713c666b4c0b00c01ad430ddb4ee8686e6b279910b6f09a3e5dR46 to use IPv4 explicitly helps.

**Comment by hashhar** [comment]
This is ready to review now. @ebyhr @nineinchnick @losipiuk

**Review by ebyhr — APPROVED** [review]

**Comment by nineinchnick** [comment]
@ebyhr CI is green, can we merge this? I need this to work one some product tests :-)

**Pull request merged**